core/src/test/java/com/opensymphony/xwork2/config/providers/EnvsValueSubstitutorTest.java

```diff
@@ -20,16 +20,55 @@
 
 import org.apache.struts2.StrutsInternalTestCase;
 
-
 public class EnvsValueSubstitutorTest extends StrutsInternalTestCase {
 
+    private boolean osIsWindows = false;  // Assume Linux/Unix environment by default
+
+    @Override
+    protected void setUp() throws Exception {
+        super.setUp();
+
+        final String os = System.getProperty("os.name");
+        if (os != null && os.startsWith("Windows")) {
+            osIsWindows = true;   // Determined that the OS is Windows (must use different environment variables)
+        }
+        else {
+            osIsWindows = false;  // Assume Linux/Unix environment by default
+        }
+    }
+
     public void testSimpleValue() throws Exception {
+
+        String expected;
+        String actual;
+        final ValueSubstitutor substitutor = new EnvsValueSubstitutor();
+
+        if (osIsWindows) {
+            // given
+            expected = System.getenv("USERNAME");
+
+            // when
+            actual = substitutor.substitute("${env.USERNAME}");
+        }
```

Comment:
I think we should aim for consistent style here and use 

```
if () {
} else {
}
```
instead of

```
if () {
}
else {
}
```

Unfortunately https://cwiki.apache.org/confluence/display/S2WIKI/Struts+2+Coding+Conventions has out-of-date links. But I see this style in most struts2 classes.

@lukaszlenart Are there any up-to-date coding conventions for struts2?

Reply:
Hello @sepe81 .

Maybe an if/else coding-conventions cleanup of these two tests would be better served with a pair of follow-up PRs that apply the convention to the two tests (along with the `commons-lang3` enhancement you identified above) both the 2.5.x and master branches ?

Yourself and @lukaszlenart can advise which convention is preferred and I'll do my best to remember.  For whatever reason, I seem to default to lining up the if-else elements column-wise, even if the { } positioning varies.  :)  :)

Reply:
see https://github.com/apache/struts/pull/272

Reply:
👍 I would like an automatic style checker tool like [apache/commons-lang](https://github.com/apache/commons-lang/blob/a668a4798163cbf2ca33584270a439369d33fedc/pom.xml#L620) but I'm not sure if any tool exists that supports checking only PR added code lines because it's hard to fix all project code styles.